api/controllers/HomeController.js

```diff
@@ -8,7 +8,8 @@
 module.exports = _.mapValues({
   async index (req, res) {
     const boxes = await Box.find({owner: req.user.name});
-    res.view('home/view', {boxes});
+    const prefs = await UserPreferences.find({user: req.user.name});
```

Comment:
If you replace this with `UserPreferences.findOne`, then the client can do `this.prefs.defaultPokemonVisibility` instead of `this.prefs[0].defaultPokemonVisibility`.